tests/test_interfaces.py

```diff
@@ -423,6 +423,36 @@ def test_output_format(
     assert processed_result["sample_id"].dtype == np.int64
 
 
+# Failing test for calibration when data is mapped to observables
+@pytest.mark.skip("Failing test for calibration when data is mapped to observables")
+@pytest.mark.parametrize("model_fixture", MODELS)
+@pytest.mark.parametrize("start_time", START_TIMES)
+@pytest.mark.parametrize("end_time", END_TIMES)
+@pytest.mark.parametrize("logging_step_size", LOGGING_STEP_SIZES)
+def test_calibrate_with_observables(
```

Comment:
I'm not sure that this test is necessary, as the existing `test_calibrate` will already create a test and then skip it based on the new changed to `setup_calibrate`.

Reply:
My suggestion is to remove this function, and then when we fix the bug we simple remove your addition on line 32 of this file.